### ISSUE TITLE
Unexpected argument count for ZRANGEBYSCORE.

### ISSUE THREAD

**Issue by bbaddepudi** [YugaByte/yugabyte-db#473]
```
redis.clients.jedis.exceptions.JedisDataException: ERR ZRANGEBYSCORE: Expected at most 5 arguments, found $0: 7
        at redis.clients.jedis.Protocol.processError(Protocol.java:127)
        at redis.clients.jedis.Protocol.process(Protocol.java:161)
        at redis.clients.jedis.Protocol.read(Protocol.java:215)
        at redis.clients.jedis.Connection.readProtocolWithCheckingBroken(Connection.java:340)
        at redis.clients.jedis.Connection.getBinaryMultiBulkReply(Connection.java:276)
        at redis.clients.jedis.Connection.getMultiBulkReply(Connection.java:269)
        at redis.clients.jedis.Jedis.zrangeByScore(Jedis.java:2022)
        at redis.clients.jedis.JedisCluster$85.execute(JedisCluster.java:945)
        at redis.clients.jedis.JedisCluster$85.execute(JedisCluster.java:942)
        at redis.clients.jedis.JedisClusterCommand.runWithRetries(JedisClusterCommand.java:120)
        at redis.clients.jedis.JedisClusterCommand.run(JedisClusterCommand.java:31)
        at redis.clients.jedis.JedisCluster.zrangeByScore(JedisCluster.java:947)
        at com.yahoo.ycsb.db.RedisClient.scan(RedisClient.java:163)
        at com.yahoo.ycsb.DBWrapper.scan(DBWrapper.java:159)
        at com.yahoo.ycsb.workloads.CoreWorkload.doTransactionScan(CoreWorkload.java:782)
        at com.yahoo.ycsb.workloads.CoreWorkload.doTransaction(CoreWorkload.java:640)
        at com.yahoo.ycsb.ClientThread.run(Client.java:455)
        at java.lang.Thread.run(Thread.java:745)
```

**Comment by bbaddepudi** [comment]
@hectorgcr noted that LIMIT option from ZRangeByScore is not implemented.

**Comment by rahuldesirazu** [comment]
Implemented in 2ffee64ad386b6bec92cb950293742a6972f0a1f.
